engine/installation/cloud/index.md

```diff
@@ -8,17 +8,10 @@ aliases:
 - /engine/installation/joyent/
 description: Cloud Installations
 keywords:
-- 'Docker install '
-menu:
-  main:
-    identifier: install_cloud
-    parent: engine_install
-    weight: "-60"
-title: On cloud providers
+- Docker install
+title: Install Engine in the cloud
```

Comment:
Switched to "on cloud hosts" which I see in other topics in this branch of the TOC